src/routes/balances/balances.controller.spec.ts

```diff
@@ -155,7 +156,9 @@ describe('Balances Controller (Unit)', () => {
             {
               tokenInfo: {
                 type: 'ERC20',
-                address: transactionApiBalancesResponse[1].tokenAddress,
+                address: transactionApiBalancesResponse[1].tokenAddress
+                  ? getAddress(transactionApiBalancesResponse[1].tokenAddress)
+                  : transactionApiBalancesResponse[1].tokenAddress,
```

Comment:
Type-wise, `tokenAddress` may be `0x${string}` or `null`, hence the ternary requirement.